### ISSUE TITLE
Add network switcher (Polygon)

### ISSUE THREAD

**Issue by ethedev** [yam-finance/synths-www#66]

**Comment by blokku-chan** [comment]
The switcher is in the wallet dropdown
<img width="198" alt="Screenshot 2021-10-18 at 3 30 14 PM" src="https://user-images.githubusercontent.com/70381938/137751302-7e4d9d5b-7091-4a5e-ba73-544ae5c83007.png">

**Comment by rossgalloway** [comment]
In the version I played with, these buttons acted like radio buttons (more than one could be selected) when they should act as switches (only one can be active at the same time).

**Comment by ethedev** [comment]
> In the version I played with, these buttons acted like radio buttons (more than one could be selected) when they should act as switches (only one can be active at the same time).

yes, and you mean the opposite acted as checkboxes while it should act as radio buttons, this will be pushed in the next deploy
